varcode/effect_ordering.py

```diff
@@ -52,10 +52,18 @@ def top_priority_transcript_effect(effects):
         elif priority == best_priority:
             best_effects.append(effect)
 
-    def effect_cds_len(effect):
-        """
-        Returns length of coding sequence of transcript associated with effect
-        """
-        return len(effect.transcript.coding_sequence)
-
-    return max(best_effects, key=effect_cds_len)
\ No newline at end of file
+    if any(effect.transcript.complete for effect in best_effects):
```

Comment:
All the transcripts in the else case e.g. `Incomplete`, `Noncoding`, or something else without a CDS. If we have at least one coding transcript, we should ignore all those without a CDS. Does that make sense?

(also, this is an axiom/assumption, not necessarily something well grounded in evidence. I could imagine a long ncRNA being the most important transcript of a gene)